README.MD

```diff
@@ -0,0 +1,326 @@
+BrandedToken.js
+============
+
+BrandedToken.js supports interaction with BrandedToken-contracts.
+The steps below describe the process of staking an EIP-20 token to create a BrandedToken on the value chain (assumed to be ethereum ) and minting (creating) a value-backed Utility Branded Token on the sidechain for use in applications.
+
+#### Overview of different components:
+
+1. BrandedToken: A Branded Token allows a mainstream application to create a value-backed token designed specifically for its application's context. A Branded Token implements the required and optional [EIP-20 Standard Token interface](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-20.md).
+
+2. GatewayComposer: A Gateway Composer facilitates staking value for branded tokens and minting a utility representation of those branded tokens with a Utility Branded Token through a gateway for use within the given application, thereby reducing both the number of transactions to execute and the number of contracts to call in order to stake and mint.
+
+3. UtilityBrandedToken: A Utility Branded Token, which also implements the EIP-20 Standard Token interface, is the utility token representation of the Branded Token on a sidechain. This representation is orchestrated via a gateway, comprising a Gateway contract on the same chain as the Branded Token, and a CoGateway contract on the sidechain with the Utility Branded Token.
+
+The following steps assume that nodejs and geth are installed and running.
+This code was tested with geth version: 1.7.3-stable. Other higher versions should also work.
+
+#### Basic setup
+
+1.  Install BrandedToken.js in your project using npm
+
+```bash
+$ npm install @openstfoundation/brandedtoken.js --save
+
+```
+
+2.  Sync your development machine with one of the following test environments
+
+    - https://ropsten.etherscan.io/
+    - https://kovan.etherscan.io/
+    - https://rinkeby.etherscan.io/
+
+3.  Create the following addresses and fund them with gas
+
+    - deployerAddress - address that deploys the EIP20Token, BrandedToken, UtilityBrandedToken and GatewayComposer contracts.
+    - facilitator - address that calls acceptStakeRequest and rejectStakeRequest.
+    - owner - address that calls GatewayComposer.requestStake.
+    - worker - address that calls BrandedToken.rejectStakeRequest.
+
+
+
+#### Creating a BrandedToken object
+The Branded Token object is an entry point for developer - using the BrandedToken object, a staking can be initiated.
```

Comment:
Suggest replacing " for developer - " with a semi-colon, as it is not clear to me what value that language adds.